swarm/network/kademlia.go

```diff
@@ -788,31 +758,40 @@ func (k *Kademlia) gotNearestNeighbours(peers [][]byte) (got bool, n int, missin
 // Health state of the Kademlia
 // used for testing only
 type Health struct {
-	KnowNN     bool     // whether node knows all its nearest neighbours
-	GotNN      bool     // whether node is connected to all its nearest neighbours
-	CountNN    int      // amount of nearest neighbors connected to
-	CulpritsNN [][]byte // which known NNs are missing
-	Full       bool     // whether node has a peer in each kademlia bin (where there is such a peer)
-	Hive       string
+	KnowNN           bool     // whether node knows all its neighbours
```

Comment:
would you consider the field names I suggested?
`NNAddrs, NNConns, MissingNNAddrs, MissingNNConns`
and function names like
`checkNNAddrs` and `checkNNConns`

or have a struct like

```golang
&Health{
Addrs: &NN{Size: n, Missing: [][]byte},
Conns: &NN{Size: n, Missing: [][]byte},
Saturated: true,
}
```

Reply:
I don't want to change it here and rebase again because of a few names. As I said in the channel there's a naming bug. Please

Reply:
Just to clarify; totally fine with the name changes. I'd just rather append them rather than rebase on them :)